### PR TITLE
num_tokens upgrade bug fixed in K8ssandra 1.3.1

### PR DESCRIPTION
**Pull request by johnsmartco**
Clarify Upgrade Notices that workaround is specific to 1.3.0

**What this PR does**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CHANGELOG.md updated (not required for documentation PRs)
- [ ] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

**Review by jdonenine — COMMENTED** [review]
Outside of the question I had about how to make more clear that 1.3.1 doesn't have this problem, looks good to go!  Thanks @johnsmartco

File: docs/content/en/release-notes/_index.md [hunk 1]
```diff
@@ -60,17 +60,19 @@ Operators are software extensions to Kubernetes that make use of custom resource
...
 
-{{% alert title="Important!" color="warning" %}}
-Before upgrading to K8ssandra 1.3.1, be sure to read the sections below.
-{{% /alert %}}
+Before upgrading to K8ssandra 1.3.x, be sure to read the sections below.
+
+### Upgrading to K8ssandra 1.3.x and Cassandra 4.0
 
-### Upgrading to K8ssandra 1.3.1 and Cassandra 4.0
+When you upgrade from a prior K8ssandra release to 1.3.0 or 1.3.1, the default `cassandra.version` is `4.0.0`. Specifically in the case of upgrading from a prior K8ssandra release to **K8ssandra 1.3.0 and Cassandra 4.0.0**, a changed Cassandra `num_tokens` default may prevent Cassandra 4.0 from starting. This section explains how to avoid the issue for 1.3.0 users.
```
[the remark this answers is not in the document]
Comment (johnsmartco) [reply docs/content/en/release-notes/_index.md:67]: Thanks @jdonenine. I could move up the note (the **Tip**) that's currently at L69-71 to the start of this upgrade section (new L66).  Would that help?  That tip note has "The bug described below was fixed in K8ssandra 1.3.1; see resolved issue [1029](https://github.com/k8ssandra/k8ssandra/issues/1029)."

[the remark this answers is not in the document]
Comment (johnsmartco) [reply docs/content/en/release-notes/_index.md]: Edited the text. Let me know - thanks.

**Review by jdonenine — APPROVED** [review]
Thanks for those last changes @johnsmartco, looks good to me now.

**Pull request merged**